pallets/providers/src/utils.rs

```diff
@@ -1035,6 +1035,9 @@ where
 
         match delta {
             RateDeltaParam::NewBucket => {
+                // Get the rate over that of a zero sized bucket of the bucket to add.
+                // If its size is less than the minimum for which rate starts to increase (which depends on the value proposition),
+                // then the rate over the zero size bucket is 0.
                 let bucket_rate = value_prop
                     .price_per_unit_of_data_per_block
```

Comment:
Why don't we make value propositions also be in "price per giga-unit of data, per block"?

Reply:
Good call, done!